docs/changelog/releases/0.12.3.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "0.12.3"
+description: SchemaHero 0.12.3 Changelog
+---
+
+
+## Changes
+
+- Update dependencies to the latest versions.
```

Comment:
did we actually update deps, or did we just update the base images?

Reply:
We did. Lots of dependabot merges.